tests/tests_lvm_pool_shared.yml

```diff
@@ -0,0 +1,121 @@
+---
+- name: Test LVM shared pools
+  hosts: all
+  become: true
+  vars:
+    storage_safe_mode: false
+    storage_use_partitions: true
+    mount_location1: '/opt/test1'
+    volume1_size: '4g'
+
+  tasks:
```

Comment:
```suggestion
  tasks:
    - name: Set up test environment for the ha_cluster role
      include_role:
        name: fedora.linux_system_roles.ha_cluster
        tasks_from: test_setup.yml

```

Reply:
part of what `test_setup.yml` does is to set `inventory_hostname` to `localhost` so that the single node clustering works.

Reply:
Fixed.